CityOfDetroit/bloom
# fix: add new google translate strings

## Issue

- Closes #1122 

## Description

Adds the `whatToExpectAdditionalText` to google translate, and pulls hardcoded English strings for accessibility features into the translation files and adds google translate translations for them.

## How Can This Be Tested/Reviewed?

Enable google translate, and ensure the `whatToExpectAdditionalText` field is translated.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
